server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -458,17 +458,12 @@ export class RestGitService {
                 Lumberjack.info(`Resolving ${key} from cache`, this.lumberProperties);
                 return cachedValue;
             }
-
-            // Value is not cached - fetch it with the provided function and then cache the value
-            winston.info(`Fetching ${key}`);
-            Lumberjack.info(`Fetching ${key}`, this.lumberProperties);
-            const value = await fetch();
-            this.setCache(key, value);
-
-            return value;
-        } else {
-            return fetch();
         }
+        // Value is not cached - fetch it with the provided function and then cache the value
+        winston.info(`Fetching ${key}`);
+        Lumberjack.info(`Fetching ${key}`, this.lumberProperties);
+        const value = await fetch();
+        this.setCache(key, value);
```

Comment:
This needs to return `value` at the bottom

Reply:
Yup. :) I lost it. Would add it in the separate pr. :)